src/webapp/components/modal-selector/ModalSelector.tsx

```diff
@@ -65,7 +66,7 @@ const _ModalSelector = React.memo((props: ModalSelectorProps) => {
             <ConfirmationDialog
                 open={openModal}
                 cancelText={i18n.t("Cancel")}
-                onCancel={() => setOpenModal(false)}
+                onCancel={() => modalActions.disable()}
```

Comment:
One nice thing of using useBooleanState is that the actions can be directly used, and we get immutability for free: `onCancel={modalActions.disable}`